zebra/zapi_msg.c

```diff
@@ -1426,7 +1426,9 @@ static void zread_route_add(ZAPI_HANDLER_ARGS)
 	re->flags = api.flags;
 	re->uptime = monotime(NULL);
 	re->vrf_id = vrf_id;
-	if (api.tableid && vrf_id == VRF_DEFAULT)
+	if (api.tableid &&
```

Comment:
I would just make this `if (api.tableid)`.

Reply:
if i is ok for you for vrf-lite, then it is ok for me.
thanks.